setup.py

```diff
@@ -32,7 +32,7 @@
 DEV = [
     "flake8",
     "pytest",
-    "pandas",  # previously pinned to 2.0.3
+    "pandas<2.2.0",  # previously pinned to 2.0.3
```

Comment:
add a note explaining why we're pinning it and open an issue

Reply:
[fixed](https://github.com/ploomber/jupysql/blob/e4f7ed57b684cd87d9d408815fcb2241aa4b7842/setup.py#L35)